### PR TITLE
translation: update translations, improve schema

### PR DESCRIPTION
**Pull request by iGormilhit**
* Pull translations from transifex.
* Removes unnecessary descriptions in document JSON schema.
* Turn labels into natural language in document JSON schema.
* Removes codes from the manual translation files that are translated
  through the label in the JSON schema form option.
* Copy the document JSON schema to the minimal document JSON schema.

Co-Authored-by: Igor Milhit <igor.milhit@rero.ch>

## Why are you opening this PR?

- To improve the document JSON schema, specifically the strings that needs translations.
- To remove strings that are not anymore useful on transifex (duplicate).
- To pull existing translations from transifex.

## How to test?

Check the changed files.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: rero_ils/manual_translations.txt [hunk 1]
```diff
@@ -43,11 +43,11 @@ _('ui_language_de')
 _('ui_language_it')
 
 # provisionActivity
-_('bf:Publication')
-_('bf:Manufacture')
-_('bf:Distribution')
-_('bf:Production')
-_('bf:Place')
+#_('bf:Publication')
```
Comment (rerowep) [inline rero_ils/manual_translations.txt:46]: If you add this here and yo change it in the documents JSON schema we have to translate it twice !
bf:Publication -> to translate
Publication -> to translate

Comment (iGormilhit) [reply rero_ils/manual_translations.txt]: You're right, I wanted to delete all this strings but forgot one. Thanks.

File: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json [hunk 2]
```diff
@@ -500,19 +495,19 @@
             "form": {
               "options": [
                 {
-                  "label": "bf:Publication",
+                  "label": "Publication",
```
Comment (rerowep) [inline rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:498]: see above

File: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json [hunk 3]
```diff
@@ -131,22 +130,22 @@
                   "value": "bf:Title"
                 },
                 {
-                  "label": "ParallelTitle",
+                  "label": "Parallel title",
```
[the remark this answers is not in the document]
Comment (iGormilhit) [reply rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:133]: Yeah, but I don't know how to do. We're supposed to use labels for human language, not codes. That's why I finally restored the `bf:...` codes in the `manual-translations.txt` file. For now, I think we're going like this.

The question is, do we need this one (and the other below) to be added to the manual translations?

**Pull request merged**